<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  
  <title>RSEM Brief Introduction | Hexo</title>
  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
  <meta name="description" content="RSEMRSEM is a kind of algorithm to determine the expression levels.
In TCGA, there are two analysis pipelines used to create Level 3 expression data from RNA Sequence data. The first approach used at">
<meta property="og:type" content="article">
<meta property="og:title" content="RSEM Brief Introduction">
<meta property="og:url" content="http://yoursite.com/2016/03/30/RSEM-Brief-Introduction/index.html">
<meta property="og:site_name" content="Hexo">
<meta property="og:description" content="RSEMRSEM is a kind of algorithm to determine the expression levels.
In TCGA, there are two analysis pipelines used to create Level 3 expression data from RNA Sequence data. The first approach used at">
<meta property="og:updated_time" content="2016-03-30T12:44:13.000Z">
<meta name="twitter:card" content="summary">
<meta name="twitter:title" content="RSEM Brief Introduction">
<meta name="twitter:description" content="RSEMRSEM is a kind of algorithm to determine the expression levels.
In TCGA, there are two analysis pipelines used to create Level 3 expression data from RNA Sequence data. The first approach used at">
  
    <link rel="alternate" href="/atom.xml" title="Hexo" type="application/atom+xml">
  
  
    <link rel="icon" href="/favicon.png">
  
  
    <link href="//fonts.googleapis.com/css?family=Source+Code+Pro" rel="stylesheet" type="text/css">
  
  <link rel="stylesheet" href="/css/style.css">
  

</head>

<body>
  <div id="container">
    <div id="wrap">
      <header id="header">
  <div id="banner"></div>
  <div id="header-outer" class="outer">
    <div id="header-title" class="inner">
      <h1 id="logo-wrap">
        <a href="/" id="logo">Hexo</a>
      </h1>
      
    </div>
    <div id="header-inner" class="inner">
      <nav id="main-nav">
        <a id="main-nav-toggle" class="nav-icon"></a>
        
          <a class="main-nav-link" href="/">Home</a>
        
          <a class="main-nav-link" href="/archives">Archives</a>
        
      </nav>
      <nav id="sub-nav">
        
          <a id="nav-rss-link" class="nav-icon" href="/atom.xml" title="RSS Feed"></a>
        
        <a id="nav-search-btn" class="nav-icon" title="Search"></a>
      </nav>
      <div id="search-form-wrap">
        <form action="//google.com/search" method="get" accept-charset="UTF-8" class="search-form"><input type="search" name="q" results="0" class="search-form-input" placeholder="Search"><button type="submit" class="search-form-submit">&#xF002;</button><input type="hidden" name="sitesearch" value="http://yoursite.com"></form>
      </div>
    </div>
  </div>
</header>
      <div class="outer">
        <section id="main"><article id="post-RSEM-Brief-Introduction" class="article article-type-post" itemscope itemprop="blogPost">
  <div class="article-meta">
    <a href="/2016/03/30/RSEM-Brief-Introduction/" class="article-date">
  <time datetime="2016-03-30T12:38:10.000Z" itemprop="datePublished">2016-03-30</time>
</a>
    
  </div>
  <div class="article-inner">
    
    
      <header class="article-header">
        
  
    <h1 class="article-title" itemprop="name">
      RSEM Brief Introduction
    </h1>
  

      </header>
    
    <div class="article-entry" itemprop="articleBody">
      
        <h1 id="RSEM"><a href="#RSEM" class="headerlink" title="RSEM"></a>RSEM</h1><p>RSEM is a kind of algorithm to determine the expression levels.</p>
<p>In TCGA, there are two analysis pipelines used to create Level 3 expression data from RNA Sequence data. The first approach used at TCGA relies on the RPKM method, while the second method uses MapSplice to do the alignment and RSEM to perform the quantitation.</p>
<h2 id="Advantages"><a href="#Advantages" class="headerlink" title="Advantages"></a>Advantages</h2><ol>
<li><p>A key feature unique to RSEM is the lack of the requirement of a reference genome. Instead, it only requires the user to provide a set of reference transcript sequences, such as one produced by a de novo transcriptome assembler </p>
</li>
<li><p>The major problems with previous methods are that it: (1) throws away data and produces biased estimates if “mappability” is not taken into account, (2) produces incorrect estimates for alternatively-spliced genes [16], and (3) does not extend well to the task of estimating isoform abundances. RSEM can address all of these issues. </p>
</li>
</ol>
<h2 id="Implemention-methods"><a href="#Implemention-methods" class="headerlink" title="Implemention methods"></a>Implemention methods</h2><p>A typical run of RSEM consists of just two steps. First, a set of reference transcript sequences are generated and preprocessed for use by later RSEM steps. Second, a set of RNA-Seq reads are aligned to the reference tran- scripts and the resulting alignments are used to estimate abundances and their credibility intervals. </p>
<h2 id="Output"><a href="#Output" class="headerlink" title="Output"></a>Output</h2><h3 id="rsem-genes-isoforms-results"><a href="#rsem-genes-isoforms-results" class="headerlink" title="rsem.genes/isoforms.results"></a>rsem.genes/isoforms.results</h3><p>The (first) RSEM paper explains that the program calculates two values. One represent the (estimated) number of reads that aligned to a transcript. This value is not an integer because RSEM only reports a guess of how many ambiguously mapping reads belong to a transcript/gene. This number is what the TCGA slightly misleadingly calls raw counts.</p>
<p>The scaled estimate value on the other hand is the estimated frequency of the gene/transcript amongst the total number of transcripts that were sequenced. Newer versions of RSEM call this value (multiplied by 1e6) TPM - Transcripts Per Million. It’s closely related to FPKM, as explained on the RSEM website. The important point is that TPM, like FPKM, is independent of transcript length, whereas “raw” counts are not!</p>
<h3 id="rsem-genes-isoforms-normalized-results"><a href="#rsem-genes-isoforms-normalized-results" class="headerlink" title="rsem.genes/isoforms.normalized.results"></a>rsem.genes/isoforms.normalized.results</h3><p>The <code>*.normalized_results</code> files on the other hand just contain a scaled version of the raw_counts column. The values are divided by the 75-percentile and multiplied by 1000. This should make the values a bit more comparable between experiments.</p>
<h2 id="References"><a href="#References" class="headerlink" title="References:"></a>References:</h2><p>Li B, Ruotti V, Stewart RM, Thomson JA, Dewey CN. (2010)<br>RNA-Seq gene expression estimation with read mapping uncertainty.<br>Bioinformatics. Feb 15;26(4):493-500.</p>

      
    </div>
    <footer class="article-footer">
      <a data-url="http://yoursite.com/2016/03/30/RSEM-Brief-Introduction/" data-id="cimevb5qz0002z2k7pwqzfuvr" class="article-share-link">Share</a>
      
      
    </footer>
  </div>
  
    
<nav id="article-nav">
  
  
    <a href="/2016/03/09/test/" id="article-nav-older" class="article-nav-link-wrap">
      <strong class="article-nav-caption">Older</strong>
      <div class="article-nav-title">test</div>
    </a>
  
</nav>

  
</article>

</section>
        
          <aside id="sidebar">
  
    

  
    

  
    
  
    
  <div class="widget-wrap">
    <h3 class="widget-title">Archives</h3>
    <div class="widget">
      <ul class="archive-list"><li class="archive-list-item"><a class="archive-list-link" href="/archives/2016/03/">March 2016</a></li></ul>
    </div>
  </div>


  
    
  <div class="widget-wrap">
    <h3 class="widget-title">Recent Posts</h3>
    <div class="widget">
      <ul>
        
          <li>
            <a href="/2016/03/30/RSEM-Brief-Introduction/">RSEM Brief Introduction</a>
          </li>
        
          <li>
            <a href="/2016/03/09/test/">test</a>
          </li>
        
          <li>
            <a href="/2016/03/09/hello-world/">Hello World</a>
          </li>
        
      </ul>
    </div>
  </div>

  
</aside>
        
      </div>
      <footer id="footer">
  
  <div class="outer">
    <div id="footer-info" class="inner">
      &copy; 2016 John Doe<br>
      Powered by <a href="http://hexo.io/" target="_blank">Hexo</a>
    </div>
  </div>
</footer>
    </div>
    <nav id="mobile-nav">
  
    <a href="/" class="mobile-nav-link">Home</a>
  
    <a href="/archives" class="mobile-nav-link">Archives</a>
  
</nav>
    

<script src="//ajax.googleapis.com/ajax/libs/jquery/2.0.3/jquery.min.js"></script>


  <link rel="stylesheet" href="/fancybox/jquery.fancybox.css">
  <script src="/fancybox/jquery.fancybox.pack.js"></script>


<script src="/js/script.js"></script>

  </div>
</body>
</html>